docs/extended_api/pipelines/pipeline_shared_state/constructor.md

```diff
@@ -0,0 +1,40 @@
+---
+grand_parent: Pipelines
+parent: cuda::pipeline_shared_state
+---
+
+# `cuda::pipeline_shared_state::pipeline_shared_state`
+
+```c++
+// Header `<cuda/pipeline>`.
```

Comment:
Same comment about "declared in" style as earlier.

This file needs a comment that this functionality is not a part of the github-released open-source library.